src/Microsoft.Extensions.Configuration.AzureAppConfiguration/ConfigurationClientManager.cs

```diff
@@ -275,7 +276,7 @@ private async Task RefreshFallbackClients(CancellationToken cancellationToken)
                     var targetEndpoint = new Uri($"https://{host}");
 
                     var configClient = _credential == null
-                        ? new ConfigurationClient(ConnectionStringUtils.Build(targetEndpoint, _id, _secret), _clientOptions)
+                        ? new ConfigurationClient(ConnectionStringUtils.Build(targetEndpoint, _id!, _secret!), _clientOptions)
```

Comment:
If `ConfigurationClientManager` is initialized using the constructor with `endpoints`, `_id` and `_secret` will be null here